src/app/ApplicationTemplate.Tests/ApplicationTemplate.Tests.csproj

```diff
@@ -1,17 +1,17 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
 	<PropertyGroup>
-		<TargetFramework>net472</TargetFramework>
+		<TargetFramework>net7.0</TargetFramework>
 		<IsPackable>false</IsPackable>
 		<GenerateDocumentationFile>True</GenerateDocumentationFile>
-		<LangVersion>10.0</LangVersion>
+		<LangVersion>11.0</LangVersion>
 		<IsTestProject>True</IsTestProject>
-		<DefineConstants>NETFRAMEWORK</DefineConstants>
```

Comment:
Was this used? Is there code that should be updated from `NETFRAMEWORK` to `TESTS`?

Reply:
no it wasn't used it was a suggestion because netframework didn't make sense anymore

Reply:
Yes and that change is totally fine. I was just checking :)